examples/rust/components/http-hello-world/wadm.yaml

```diff
@@ -15,9 +15,9 @@ spec:
     - name: http-component
       type: component
       properties:
-        image: file://./build/http_hello_world_s.wasm
+        # image: file://./build/http_hello_world_s.wasm
         # To use the a precompiled version of this component, use the line below instead:
-        # image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
+        image: ghcr.io/wasmcloud/components/http-hello-world-rust:0.1.0
```

Comment:
Heads up that this was a case were we referred to the local image and not the OCI ref. I think this one was just accidentally missed but giving it a shoutout in case I'm wrong.

Reply:
This should be the local ref because it's what we use in our tutorials, and our tutorials go right into modifying the wasm

Reply:
TY, added a note. We'll need to use a different example for the operator quickstart